package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@transform-data/transform-react",
-  "version": "1.14.0",
+  "version": "1.14.1",
```

Comment:
Since it's a non-breaking change, this seemed appropriate; happy to go with a `1.15.0` instead, if you'd prefer, @jjrajani!

Reply:
I think we should go with 1.15.0 https://docs.npmjs.com/about-semantic-versioning